mycrypt.py

```diff
@@ -0,0 +1,500 @@
+#!/usr/bin/env python
+#!python
+'''
+Implement openssl compatible AES-256 CBC mode encryption/decryption.
+
+This module provides encrypt() and decrypt() functions that are compatible
+with the openssl algorithms.
+
+This is basically a python encoding of my C++ work on the Cipher class
+using the Crypto.Cipher.AES class.
+
+URL: http://projects.joelinoff.com/cipher-1.1/doxydocs/html/
```

Comment:
Having a library that is commonly used and installed/updated via `pip` would make me feel much better. The URL of this project doesn't even work anymore. You might have tried this already but I'd recommend to use a library that has bindings to openssl itself. I wouldn't be picky if this isn't about encryption. 

![](https://cloud.githubusercontent.com/assets/32488/16747177/24792662-47be-11e6-850b-4820962e5e46.gif)